dspace-api/src/main/java/org/dspace/versioning/dao/VersionDAO.java

```diff
@@ -24,4 +26,10 @@
 public interface VersionDAO extends GenericDAO<Version>
 {
     public Version findByItem(Context context, Item item) throws SQLException;
+    
+    public List<Version> findVersions(Context context, VersionHistory versionHistory)
```

Comment:
We might want to rename this method to make it more clear that only versions with an item attached to it are returned.